src/styles/blocks/about.scss

```diff
@@ -0,0 +1,45 @@
+.about {
+  margin: 120px 0 110px;
+
+  @include onTablet {
+    margin-bottom: 96px;
+  }
+
+  @include onDesktop {
+    margin-bottom: 120px;
+  }
+
+  &__title {
+    @include title;
+    margin: 0 0 32px;
+    text-align: left;
```

Comment:
It's better to use a consistent text alignment. If you want to align the title to the left in this specific case, consider using a modifier class like .about__title--left instead of changing the base style. 😇